### PR TITLE
misc changes to ClusterGenerator of dataproc

### PR DESCRIPTION
**Pull request by prakash12**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/providers/google/cloud/operators/dataproc.py [hunk 1]
```diff
@@ -678,7 +680,7 @@ class DataprocScaleClusterOperator(GoogleCloudBaseOperator):
                 graceful_decommission_timeout='1h',
                 dag=dag)
 
-    .. seealso::
+    .. see also::
```
Comment (pankajastro) [inline airflow/providers/google/cloud/operators/dataproc.py]: `.. seealso::` looks correct me

**Review by pankajastro — COMMENTED** [review]
can you please add some description about changes?

File: airflow/providers/google/cloud/operators/dataproc.py (hunk 1 above)
Comment (prakash12) [reply airflow/providers/google/cloud/operators/dataproc.py]: Reverted.

**Comment by prakash12** [comment]
Description added.
This PR contains changes to improve code quality. I have declared a variable for string which are being used multiple times.

**Review by potiuk — APPROVED** [review]
Insignificant changes, but yeah. Any improvement is good.

**Pull request merged**